### ISSUE TITLE
@Import triggers Uncaught TypeError: Arguments to path.join must be strings

### ISSUE THREAD

**Issue by furins** [AtomLinter/linter-less#13]
Apparently it seems that the parser does not take in account for [lesscss import options](http://lesscss.org/features/#import-options) and assumes the second token to be a path, even if in brackets.

I started to see this error only yesterday, so it may be related to recent changes.

### Steps to reproduce

edit a LESS file that has an `@import (inline)` statement like this:

    @import (inline) "./icons.css";

il you omit the `(inline)` part, the error is not triggered.

### Stack Trace

Uncaught TypeError: Arguments to path.join must be strings

```
At path.js:449

TypeError: Arguments to path.join must be strings
  at Object.posix.join (path.js:449:13)
  at tryPathIndex (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/less-cache/node_modules/less/lib/less/index.js:230:37)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/less-cache/node_modules/less/lib/less/index.js:233:29
  at FSReqWrap.oncomplete (fs.js:77:15)

```

### Commands

```
     -6:00.2.0 core:save (atom-text-editor.editor.is-focused)
     -5:24.9.0 core:backspace (atom-text-editor.editor.is-focused)
  4x -5:20.9.0 core:save (atom-text-editor.editor.is-focused)
     -3:32.5.0 core:cut (atom-text-editor.editor.is-focused)
     -3:30.2.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -3:30.1.0 editor:newline (atom-text-editor.editor.is-focused)
     -3:30 core:paste (atom-text-editor.editor.is-focused)
     -3:29.1.0 core:save (atom-text-editor.editor.is-focused)
  3x -1:31 core:backspace (atom-text-editor.editor.is-focused)
     -1:03.5.0 core:save (atom-text-editor.editor.is-focused)
     -0:40.7.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:40.7.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -0:39.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:36.9.0 core:save (atom-text-editor.editor.is-focused)
     -0:01.7.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:01.7.0 editor:newline (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".DS_Store",
      "*.pyc"
    ],
    "disabledPackages": [
      "wrap-guide",
      "auto-indent",
      "timecop",
      "gulp-snippets"
    ],
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  },
  "editor": {
    "fontFamily": "\"input mono\"",
    "fontSize": 11,
    "showInvisibles": true,
    "showIndentGuide": true,
    "tabLength": 4,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.0
angularjs-snippets, v0.10.17
atom-beautify, v0.28.6
auto-update-packages, v1.0.0
color-picker, v2.0.7
django-templates, v0.5.0
emmet, v2.3.10
file-icons, v1.5.8
hex, v0.6.2
language-jade, v0.4.0
language-restructuredtext, v0.6.0
linter, v1.0.5
linter-csslint, v0.0.13
linter-eslint, v1.0.8
linter-less, v2.0.0
linter-php, v0.0.15
linter-pyflakes, v0.0.5
minimap, v4.10.0
python-isort, v0.0.7

# Dev
No dev packages
```

### System information

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: Atom Core

**Comment by josa42** [comment]
Thx for reporting this. It should be fixed in the latest linter-less release (2.0.3).

**Comment by furins** [comment]
Thank you for the prompt reply/fix!
